File: schedule-web/src/types/reminder.ts
export interface Reminder {
  id: number;
  userId: number;
  taskId: number;
  remindTime: string;
  remindType: string;
  channel: string;
  message: string;
  status: string;
  createdAt: string;
}

export interface ReminderUpdateDTO {
  remindTime?: string;
  remindType?: string;
  channel?: string;
  message?: string;
}

File: schedule-server/src/main/java/com/aspire/schedule/repository/entity/Reminder.java
package com.aspire.schedule.repository.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.time.LocalDateTime;

@Data
@TableName("reminder")
public class Reminder {

    @TableId(type = IdType.AUTO)
    private Long id;
    private Long userId;
    private Long taskId;
    private LocalDateTime remindTime;
    private String remindType;
    private String channel;
    private String message;
    private String status;
    private LocalDateTime createdAt;
}
